### DISCUSSION TITLE
AutoUpdate

### DISCUSSION THREAD

**Discussion by Rograr** [Storik4pro/goodbyeDPI-UI#34] [Ideas]
Сделайте возможность автоматического обновления, когда оно появляется. 
У меня уведомлений о новой версии нет, а в настройки приложения не захожу. Было бы удобно, если бы оно само обновлялось и перезапускалось без моего участия. 

______

Make it possible to automatically update when it appears. 
I don’t have notifications about the new version, and I don’t go into the application settings. It would be convenient if it updated and restarted itself without my participation.

**Comment by Storik4pro** [comment]
Хорошо, попробую реализовать
